app/build.gradle

```diff
@@ -48,8 +48,8 @@ allprojects {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    //noinspection GradleCompatible
-    implementation 'androidx.appcompat:appcompat:1.1.0'
+    //use 1.0.2 version because in 1.1.0 version WebView crashes in API 21
```

Comment:
Sometimes there is auto-formatting hapenning, which sorts comments and dependencies. So they can be split. Thus it's better to write such comments on the same line as dependency.

Reply:
Using `/**/` ?